<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>flitWidget</class>
 <widget class="QWidget" name="flitWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>340</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Send Flit via SystemIF</string>
  </property>
  <widget class="QLabel" name="dataLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Data (hex):</string>
   </property>
  </widget>
  <widget class="QLabel" name="addressLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Address (hex):</string>
   </property>
  </widget>
  <widget class="QLabel" name="routeLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Route Info:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addressEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>Selects the local address of the flit recipient...</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>Data to write / Read Result</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QComboBox" name="subDestIDBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>81</width>
     <height>29</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>Selects the destination Sub-ID...</string>
   </property>
   <item>
    <property name="text">
     <string>CRB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MPB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RTG</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CORE0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CORE1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PERIE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PERIS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PERIW</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PERIN</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>301</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="readButton">
      <property name="whatsThis">
       <string>Read from selected Destination...</string>
      </property>
      <property name="text">
       <string>Read</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="writeButton">
      <property name="whatsThis">
       <string>Write to selected Destination...</string>
      </property>
      <property name="text">
       <string>Write</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="doneButton">
      <property name="whatsThis">
       <string>Quit this dialog...</string>
      </property>
      <property name="text">
       <string>Done</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="addressBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>81</width>
     <height>29</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>Selects predefined CRB addresses...</string>
   </property>
   <item>
    <property name="text">
     <string>GLCFG0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GLCFG1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>L2CFG0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>L2CFG1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SENSOR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GCBCFG</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TILEID</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LOCK0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LOCK1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CUSTOM</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="routeBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>Selects the destination tile...</string>
   </property>
   <item>
    <property name="text">
     <string>Tile x=0, y=0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=1, y=0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=2, y=0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=3, y=0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=4, y=0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=5, y=0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=0, y=1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=1, y=1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=2, y=1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=3, y=1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=4, y=1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=5, y=1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=0, y=2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=1, y=2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=2, y=2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=3, y=2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=4, y=2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=5, y=2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=0, y=3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=1, y=3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=2, y=3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=3, y=3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=4, y=3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=5, y=3</string>
    </property>
   </item>
  </widget>
  <widget class="QRadioButton" name="radioButtonHex">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>96</y>
     <width>78</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Hex Data</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonBin">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>111</y>
     <width>97</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Bin Data</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>routeBox</tabstop>
  <tabstop>subDestIDBox</tabstop>
  <tabstop>addressEdit</tabstop>
  <tabstop>addressBox</tabstop>
  <tabstop>dataEdit</tabstop>
  <tabstop>readButton</tabstop>
  <tabstop>writeButton</tabstop>
  <tabstop>doneButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>doneButton</sender>
   <signal>clicked()</signal>
   <receiver>flitWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
